nvflare/private/fed/client/communicator.py

```diff
@@ -194,8 +196,8 @@ def pull_task(self, servers, project_name, token, ssid, fl_ctx: FLContext):
         return_code = task.get_header(MessageHeaderKey.RETURN_CODE)
 
         if return_code == ReturnCode.OK:
-            size = len(task.payload)
-            task.payload = task.payload
+            size = asizeof.asizeof(task.payload)
```

Comment:
Maybe this is okay here since task.payload is already serialized into bytes or list of bytes...

Reply:
Now think of it, I'm not sure this is the right func to use. According to its documentation, this func computes the bytes used for the python object. But we are interested in the size of the message sent. Maybe this func can do it, but if the payload is just bytes or list of bytes, we should be able compute the size easily.